sixtyfps_runtime/rendering_backends/gl/lib.rs

```diff
@@ -1029,20 +926,17 @@ impl ItemRenderer for GLItemRenderer {
         {
             let paint = layer_to_restore.image.as_paint();
 
-            self.shared_data
-                .layer_images_to_delete_after_flush
-                .borrow_mut()
-                .push(layer_to_restore.image);
+            self.layer_images_to_delete_after_flush.push(layer_to_restore.image);
```

Comment:
Removing this use of interior mutability is what triggers the change from `FnOnce` to `FnMut`. If you prefer then I can keep that vector in a RefCell instead and stay with `FnOnce`.

Reply:
In the forest of warnings I overlooked the one that really caused it, which was an immutable borrow of self outside the closure, that didn't go well with the mixture of immutable and mutable borrows inside. But fortunately that was easy to solve :)